webui/react/src/constants/access-control.ts

```diff
@@ -0,0 +1 @@
+export const OMITTED_STR = '***';
```

Comment:
**nit:** I noticed some files have broken this pattern, but camelcase filename for consistency? `accessControl.ts`